app/assets/javascripts/discourse/app/models/topic.js

```diff
@@ -256,6 +257,19 @@ const Topic = RestModel.extend({
 
   @discourseComputed("last_read_post_number", "highest_post_number", "url")
   lastUnreadUrl(lastReadPostNumber, highestPostNumber) {
+    let customUrl = null;
+    _customLastUnreadUrlCallbacks.some((cb) => {
```

Comment:
I think some form of test coverage here will be useful for the future. Perhaps a unit tests for just the computed property?

Reply:
Added an acceptance test, found it easier to do than a unit test, and it tests the functionality a tiny bit more generally.